packages/SwingSet/src/controller.js

```diff
@@ -157,12 +159,23 @@ export async function makeSwingsetController(
     slogFile && (await fs.createWriteStream(slogFile, { flags: 'a' })); // append
 
   function writeSlogObject(obj) {
-    // TODO sqlite
+    if (!slogSender && !slogF) {
+      // Fast path; nothing to do.
+      return;
+    }
+
+    // Add a timestamp to the slog input object.
+    const timeMS = performance.timeOrigin + performance.now();
+    const time = timeMS / 1000;
+    const timedObj = { time, ...obj };
```

Comment:
We don't have much of a schema for this, but the loadgen relies on `time` and `type` field appearing before anything else, in that order.

One way to ensure the `time` does not get overridden, yet comes first before `type` would be:

```suggestion
    const timedObj = { time: undefined, type: undefined, ...obj, time };
```